packages/wheelhouse-core/src/kubernetes/kubernetesActions.js

```diff
@@ -0,0 +1,32 @@
+import { KUBERNETES_DATA } from "./kubernetesConstants";
+
+export const kubernetesStartPullingData = () => dispatch => {
+  setInterval(() => {dispatch(kubernetesData("get", "pods"));}, 5000);
+};
+
+export const kubernetesData = (action, resource) => dispatch => {
+  const spawn = require("child_process").spawn;
+  const ls = spawn("kubectl", [action, resource, "-o", "json"]);
```

Comment:
This variable name is an artifact of the example we copied -- a better one for us would be `kubectl`